lib/beaker-answers/pe_conf.rb

```diff
@@ -82,6 +82,28 @@ def _generate_1_0_data
         pe_conf["#{ns}::puppetdb_host"] = puppetdb.hostname
       end
 
+      if any_host_with_role?('pe_postgres')
+        pe_conf["#{ns}::database_host"] = the_host_with_role('pe_postgres').hostname
+        if hosts[0].options[:pe_postgresql_options]
```

Comment:
are the pe_postgresql_options being set at the global level, or are you going to set them per host? If per host, is there a reason to have it on the host level instead of global?

Reply:
Good question. When I'm supplying an options file, Beaker's global options is empty, the options are set at the host level.

Reply:
Am I doing something wrong there? Seems odd to me.